metabase/metabase
# feat(sdk): ability to enforce the destination collection to save to and hide the collection picker

Closes https://github.com/metabase/metabase/issues/49193

### Description

Provides the ability to use `saveOptions.saveToCollectionId` to save the collection to a specified collection id, while hiding the collection picker from the save modal.

### How to verify

- Go to Storybook: http://localhost:6006/?path=/story/embeddingsdk-interactivequestion--default
- Set `isSaveEnabled` to `true` and `saveToCollectionId` to a collection id of choice
- Click on Save
- Click on "Save as new question"
- The collection picker should not be visible now
- Click on Save
- The network request should indicate that you are indeed saving to that `collectionId`

### Checklist

- [x] Tests have been added/updated to cover changes in this PR - E2E test
